### PR TITLE
Update hubble e2e example to create visualization tutorial

### PR DESCRIPTION
**Pull request by chowbao**
Update hubble e2e example to create visualization tutorial

* Rename to `Creating Visualizations Tutorial`
* Added final result gif/image and expectation at top of page
* Updated images to be even clearer in what should be clicked/done

### CODE REVIEW

File: docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ---
-title: "End to End Analysis Example"
+title: "Creating Visualizations Tutorial"
```
Comment (sydneynotthecity) [inline docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx:2]: nit: I would drop `Tutorial`

Maybe in future we add a Tutorials section this would go under

File: docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx [hunk 2]
```diff
@@ -22,7 +24,14 @@ import filter_contract from "/img/hubble/filter-contract.png";
 import persistent_only from "/img/hubble/persistent-only.png";
 import filtered_report from "/img/hubble/filtered-report.png";
 
-This page will guide you through an end-to-end analysis use case with Hubble and [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en)
+This page will show how you can create visualizations with Hubble and [Google Looker Studio](https://cloud.google.com/looker-studio?hl=en).
+
+By the end of the tutorial you will have created two graphs that will help visualize temporary VS persistent contract data entry distribution and expiration.
+
+<img src={hubble_visualization_demo} width="1000" />
```
Comment (sydneynotthecity) [inline docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx:31]: Love the gif, it really shows the end user how they can customize their visuals. 
wdyt about dropping the `hubble_visualization_final`? it's kinda weird to me to include both at the top, feels duplicative

File: docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx [hunk 3]
```diff
@@ -22,7 +24,14 @@ import filter_contract from "/img/hubble/filter-contract.png";
...
+<img src={hubble_visualization_final} width="500" />
+
+As you can see, persistent contract data entries account for roughly 25% of all contract data entries with the rest being temporary. There are also a lot more expired temporary contract data entries. This is expected because temporay entries cannot be bumped whereas persistent entries can be bumped/restored. Please see [State Archival - Contract Data Type Descriptions](https://developers.stellar.org/docs/learn/encyclopedia/storage/state-archival#contract-data-type-descriptions) for more information.
```
Comment (sydneynotthecity) [inline docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx:34]: nit: I would reword the last sentence: 
> For more information, you can read about [State Archival and the Contract Data Lifecycle].

File: docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx [hunk 4]
```diff
@@ -137,7 +146,7 @@ order by 1 desc, 2
 
 :::note
 
-Note that between `2024-02-01` and `2024-06-30` there aren't that many expired persistent entries
+Note that between `2024-02-01` and `2024-10-31` there aren't that many expired persistent entries
```
Comment (sydneynotthecity) [inline docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx:149]: Why does this matter? Curious what we're trying to highlight with the note

**Review by sydneynotthecity — COMMENTED** [review]
I would also update the `Create a Dashboard Analyzing Soroban Metrics` to something like `Create a Report in Looker Studio`. That will give us the flexibility to either add sections for different tooling (Dune? python libraries? etc) while keeping the tutorial generic.

Also review Step #5 under `Use Custom SQL...` the markdown formatting is funky and blocks the incorrect words.

Lastly, should step #7 onward be its own section? `Applying a Global Filter` or something similar. These steps context switch without any transition which is confusing

File: docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx (hunk 2 above)
Comment (chowbao) [reply docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx:31]: Hmm yeah I'll drop it

File: docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx (hunk 4 above)
Comment (chowbao) [reply docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx:149]: I was trying to warn that the differences between temp and persistent expiration is pretty large. But yeah that point isn't necessary for this tutorial

File: docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx (hunk 1 above)
Comment (chowbao) [reply docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx]: updated in c560544b37fe5785fb8eac095e6156ecc457b4e0

File: docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx (hunk 3 above)
Comment (chowbao) [reply docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx]: updated in c560544b37fe5785fb8eac095e6156ecc457b4e0

File: docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx (hunk 4 above)
Comment (chowbao) [reply docs/data/hubble/analyst-guide/creating-visualizations-tutorial.mdx]: updated in c560544b37fe5785fb8eac095e6156ecc457b4e0

**Review by sydneynotthecity — APPROVED** [review]
